### DISCUSSION TITLE
Using Global Plugins?

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#237] [Q&A] [answered]
**Malt 1.0.0.beta**
I copied the `PluginExample` folder from the repository and under Malt preferences I selected that copied directory as my path. Shouldn't I be able to use the `instance_random_color()` function in the `PluginExample.glsl` file now? Because I cannot find it. I know you are still planning on documenting all this but I was wondering if there was something obvious that I missed here.

**Comment by pragma37** [comment, marked as answer]
Your Global Plugins path should not point to the PluginExample itself, it should point to its parent folder.
So you should create a Plugins folder and select that path, then any plugin that you drop into that folder should be automatically loaded.

**Reply by Kolupsy** [reply]
oooh i see. and the Plugins folder doesnt need an \_\_init\_\_.py file then?

UPDATE: no it didnt need an init file but i had to restart blender to be able to add that instance_random_color node

**Reply by pragma37** [reply]
> no it didnt need an init file but i had to restart blender to be able to add that instance_random_color node

You can also manually reload Malt:
![imagen](https://user-images.githubusercontent.com/36610999/152821194-6fb30f5f-3c1a-434b-9a80-c7792999afff.png)
However, once a plugin has been loaded, node libraries should auto-update whenever you modify them.

**Reply by Kolupsy** [reply]
Yes, that I can confirm! Also the reload button works fine.
